### PR TITLE
feat: add ucore nvidia blog post

### PR DESCRIPTION
**Pull request by bsherman**

### CODE REVIEW

File: docs/blog/posts/ucore-nvidia.md [hunk 1]
```diff
@@ -0,0 +1,79 @@
+---
+date: 2023-10-05
+comments: true
+authors: 
+  - bsherman
+links:
+  - ucore Commits: https://github.com/ublue-os/ucore/commits/main
+---
+# Universal Core("ublue" for the server) Gains NVIDIA Support
```
Comment (castrojo) [inline docs/blog/posts/ucore-nvidia.md:9]:
```suggestion
# Universal Core Gains NVIDIA Support

Universal Core is a Universal Blue custom image of Fedora CoreOS. 
```

File: docs/blog/posts/ucore-nvidia.md [hunk 2]
```diff
@@ -0,0 +1,79 @@
...
+
+While Universal Blue has rightly been receiving much attention on the desktop; [Universal Core (ucore)](https://github.com/ublue-os/ucore) has been quietly growing its own set of features for the server side of things.
```
Comment (castrojo) [inline docs/blog/posts/ucore-nvidia.md:11]:
```suggestion
While Universal Blue has been receiving much attention on the desktop; [Universal Core (ucore)](https://github.com/ublue-os/ucore) has been quietly growing its own set of features for the server side of things.
```

File: docs/blog/posts/ucore-nvidia.md [hunk 3]
```diff
@@ -0,0 +1,79 @@
...
+
+## Announcing NVIDIA Support
+
+Today we get to announce a big feature indeed: **ucore finally has nvidia drivers built in!** *(optionally built-in, of course, not everyone wants/needs it)*
```
Comment (castrojo) [inline docs/blog/posts/ucore-nvidia.md:15]:
```suggestion
Today we get to announce a long awaited feature:  **ucore finally has images available nvidia drivers built in!** 
```

File: docs/blog/posts/ucore-nvidia.md [hunk 4]
```diff
@@ -0,0 +1,79 @@
...
+
+As one of our [oldest feature requests](https://github.com/ublue-os/ucore/issues/18), which also happend to be created on April 1st, some probably wondered if it was an April Fools Day joke. But no, 6 months later, the feature has landed.
+
+### NVIDIA for a Homelab
+
+What could this feature provide a CoreOS/ucore user with an NVIDIA card?
+
+Typically its going to be one of:
+- [transcoding video](https://developer.nvidia.com/video-encode-and-decode-gpu-support-matrix-new) is quite popular, especially if you don't have a later gen Intel CPU with QuickSync
+- [CUDA](https://developer.nvidia.com/cuda-gpus) for AI, or other GPU accelerated workloads
```
Comment (castrojo) [inline docs/blog/posts/ucore-nvidia.md:25]:
```suggestion
Typically its going to be one of:
- [CUDA](https://developer.nvidia.com/cuda-gpus) for AI, or other GPU accelerated workloads
- [transcoding video](https://developer.nvidia.com/video-encode-and-decode-gpu-support-matrix-new) is quite popular, especially if you don't have a later gen Intel CPU with QuickSync

```

I would put this one first. 😄

File: docs/blog/posts/ucore-nvidia.md [hunk 5]
```diff
@@ -0,0 +1,79 @@
...
+
+[FrigateNVR](https://frigate.video) is a great example of a self-hostable open source project which can use both [video transcode](https://docs.frigate.video/configuration/hardware_acceleration#nvidia-gpus) capabilities and [compute](https://docs.frigate.video/configuration/detectors#nvidia-tensorrt-detector) capabilities to power a smart home camera solution.
+
+
+### Details
+
+So, what's actually included with this "nvidia release"?
```
Comment (castrojo) [inline docs/blog/posts/ucore-nvidia.md:32]:
```suggestion
So, what's actually included with this "Nvidia release"?
```

File: docs/blog/posts/ucore-nvidia.md [hunk 6]
```diff
@@ -0,0 +1,79 @@
...
+
+1. [nvidia kernel driver](https://negativo17.org/nvidia-driver)
```
Comment (castrojo) [inline docs/blog/posts/ucore-nvidia.md:34]:
```suggestion
1. [Nvidia kernel driver](https://negativo17.org/nvidia-driver)
```

File: docs/blog/posts/ucore-nvidia.md [hunk 7]
```diff
@@ -0,0 +1,79 @@
...
+    - We pre-build the [latest nvidia driver (kmod)](https://github.com/negativo17/nvidia-driver/blob/master/nvidia-driver.spec) from [negativo17's fedora-nvidia repository](https://negativo17.org/repos/fedora-nvidia.repo).
+    - This upstream was chosen over rpmfusion due to its granular packaging concept which allows the installation of only minimal `nvidia-driver-cuda` packages and does NOT require xorg/wayland/etc as dependencies.
+2. [nvidia-container-toolkit](https://docs.nvidia.com/datacenter/cloud-native/container-toolkit/latest/sample-workload.html)
+    - CoreOS is focussed on container workloads, and to use NVIDIA on containers, you need their toolkit.
```
Comment (castrojo) [inline docs/blog/posts/ucore-nvidia.md:38]:
```suggestion
    - CoreOS is focussed on container workloads, and to use Nvidia on containers, you need their toolkit.
```

File: docs/blog/posts/ucore-nvidia.md [hunk 8]
```diff
@@ -0,0 +1,79 @@
...
+    - The latest version is installed here which simultaneously supports both root and rootless containers. 
+    - Both podman and docker can be used.
+3. [nvidia container selinux policy](https://github.com/NVIDIA/dgx-selinux/tree/master/src/nvidia-container-selinux)
+    - NVIDIA's toolkit docs give examples of running with SELinux disabled on the container: `podman run --security-opt label=disable --device=nvidia.com/gpu=all ubuntu nvidia-smi` but some users may want more security.
```
Comment (castrojo) [inline docs/blog/posts/ucore-nvidia.md:42]:
```suggestion
3. [Nvidia container selinux policy](https://github.com/NVIDIA/dgx-selinux/tree/master/src/nvidia-container-selinux)
    - Nvidia's toolkit docs give examples of running with SELinux disabled on the container: `podman run --security-opt label=disable --device=nvidia.com/gpu=all ubuntu nvidia-smi` but some users may want more security.
```

File: docs/blog/posts/ucore-nvidia.md [hunk 9]
```diff
@@ -0,0 +1,79 @@
...
+`ucore` images add the following to CoreOS:
+  - [cockpit](https://cockpit-project.org) - GUI for managing the server
+  - [distrobox](https://github.com/89luca89/distrobox)
+  - guest VM agents (`qemu-guest-agent` and `open-vm-tools`)
+  - moby-engine(docker), docker-compose and podman-compose
+  - [tailscale](https://tailscale.com) and [wireguard-tools](https://www.wireguard.com)
+  - [tmux](https://github.com/tmux/tmux/wiki/Getting-Started)
+  - *optionally*
+    - [NVIDIA](#details)
+    - [ZFS](https://openzfs.github.io/openzfsdocs/Getting%20Started/Fedora/index.html)
```
Comment (castrojo) [inline docs/blog/posts/ucore-nvidia.md:65]:
```suggestion
  - *Optionally*
    - [Nvidia](#details)
    - [ZFS](https://openzfs.github.io/openzfsdocs/Getting%20Started/Fedora/index.html)
```

File: docs/blog/posts/ucore-nvidia.md [hunk 10]
```diff
@@ -0,0 +1,79 @@
...
+
+`ucore-hci` builds upon `ucore` to provide "hyper-converged infrastructure". These images add more hardware/driver support, more storage tools, and virtualization(hypervisor) support:
+  - [cockpit-machines](https://github.com/cockpit-project/cockpit-machines): Cockpit GUI for managing virtual machines
+  - intel wifi firmware - CoreOS omits this despite including atheros wifi firmware
```
Comment (castrojo) [inline docs/blog/posts/ucore-nvidia.md:69]:
```suggestion
  - Intel WiFi firmware - CoreOS omits this despite including Atheros WiFi firmware
```

**Review by castrojo — CHANGES_REQUESTED** [review]

**Review by castrojo — CHANGES_REQUESTED** [review]

File: docs/blog/posts/ucore-nvidia.md (hunk 3 above)
Comment (castrojo) [reply docs/blog/posts/ucore-nvidia.md:15]:
```suggestion
Today we get to announce a long awaited feature:  **ucore finally has images available with Nvidia drivers built in!** 
```

File: docs/blog/posts/ucore-nvidia.md (hunk 9 above)
Comment (bsherman) [reply docs/blog/posts/ucore-nvidia.md]: partially applied

**Review by castrojo — APPROVED** [review]

**Pull request merged**